datasets/new_dataset.py

```diff
@@ -17,10 +22,37 @@
     ConformerGenerationError,
 )
 from openff.toolkit.utils.toolkits import OpenEyeToolkitWrapper
+from qcportal import PortalClient
 from tqdm import tqdm
+from yammbs.inputs import QCArchiveDataset
 
+logging.basicConfig(level=logging.INFO)
 logger = logging.getLogger(__name__)
-logger.setLevel(logging.INFO)
+
+
+@dataclass
+class Config:
+    ds_name: str
```

Comment:
Are these the only two arguments for config or do you anticipate adding more in the future? If so, would it be a good idea to add a version argument to distinguish between config generations? (If not -- would it be easier to pass the name and chunksize as arguments to argparse?)

Reply:
IMO it would be easier to just pass the dataset name and chunksize to argparse

Reply:
Yes, I was anticipating more options in the future, but to avoid this confusion I will just move everything into argparse.